uberduck_ml_dev/data/parse.py

```diff
@@ -0,0 +1,163 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/data.parse.ipynb (unless otherwise specified).
+
+__all__ = ['CACHE_LOCATION', 'STANDARD_MULTISPEAKER', 'STANDARD_SINGLESPEAKER', 'VCTK']
+
+# Cell
+import os
+import numpy as np
+import sqlite3
+import uuid
+from pathlib import PosixPath
+from tqdm import tqdm
+from pathlib import Path
+import os
+import pandas as pd
+import json
+
+CACHE_LOCATION = Path.home() / Path(".cache/uberduck/uberduck-ml-exp.db")
```

Comment:
Can we use `uberduck_ml_dev.data.cache.CACHE_LOCATION` here rather than defining the variable in two places?

Reply:
Sure, but the variables stored in this cache are uberduck specific and so I'm not sure where default should be (perhaps not in uberduck_ml_dev?)

Reply:
The location in this PR is generic (`~/.cache/uberduck/uberduckk-ml-exp.db`). The previous location was also generic (`~/.cache/uberduck-uberduck-ml-dev.db`). Actually I think the old path is better for the reason you state (it makes more sense to give the cache db the same name as this repo)